less/sortableTable.less

```diff
@@ -48,9 +23,6 @@ table.sortableTable {
         content: "\f078";
       }
     }
-    th + th {
-      border-left: 1px solid @braveOrange;
-    }
```

Comment:
Converted to:
https://github.com/brave/browser-laptop/pull/10454/files#diff-159ff36704e059402c3f17fb8a35a6fcR502
https://github.com/brave/browser-laptop/pull/10454/files#diff-159ff36704e059402c3f17fb8a35a6fcR518